src/libdrakvuf/private.h

```diff
@@ -236,7 +236,7 @@ struct memcb_pass {
     drakvuf_t drakvuf;
     uint64_t gfn;
     addr_t pa;
-    char *procname;
+    proc_data_t *proc_data ;
     int64_t userid;
```

Comment:
Is `userid` still needed here or would `proc_data` have it already?

Reply:
```proc_data``` have it but I don't know if it's still needed, I'll take a look

Reply:
You're right, it's not needed 👍